templates/web-app/moon.yml

```diff
@@ -0,0 +1,8 @@
+tasks:
+  dev:
+    env:
+      DOMAIN: "$project.${TS_DOMAIN}"
+    deps:
+      - target: "privi:dev"
+      - target: "funnel:start"
+        args: ["5173", "$project"]
```

Comment:
Lets use a port far away from 5173 (6000 might be good). 5173 is the same as our base frontend, and there are probably testing scenarios where we want an app frontend and the Habitat frontend running together. 5174 is not a good option because vite will increment the port number, so its easy for there to be conflicts all over the place